File: ios/AdmobNative.m
#import <React/RCTBridgeModule.h>

@interface RCT_EXTERN_MODULE(AdmobNative, NSObject)

RCT_EXTERN_METHOD(loadAd:(RCTPromiseResolveBlock)resolve
				  withRejecter:(RCTPromiseRejectBlock)reject)

@end

File: ios/AdmobNative.swift


@objc(AdmobNative)
class AdmobNative: NSObject {

    @objc(withResolver:withRejecter:)
    func load(resolve:RCTPromiseResolveBlock,reject:RCTPromiseRejectBlock) -> Void {
		var admobDelegate = AdmobDelegateResolver(adUnitID: "", resolve: resolve, reject: reject)
		admobDelegate.load()
    }
}
